src/client/css/style.css

```diff
@@ -475,6 +475,12 @@ section#events {
   bottom: unset;
 }
 
+.cs-style-3 figure:hover figcaption span
+{
```

Comment:
Nit. Please keep brackets on same line with single space between selector and `{`.